tools/json_tools/table.py

```diff
@@ -61,7 +61,7 @@ def safe_value(value, format):
 
     """
     if format == 'md':
-        return value.replace('|', '\|')
+        return value.replace('|', '\\|')
```

Comment:
@wapcaplet Can you please take a look if PEP8 was wrong here?

Reply:
It appears to work either way, but the `\\|` version seems more correct to me.

>>> '|'.replace('|', '\|')
'\\|'
>>> '|'.replace('|', '\\|')
'\\|'